requirements.txt

```diff
@@ -11,3 +11,30 @@
 
 # now run the normal setup.py and get all the normal PyPI dependencies from there
 -e .
+
+boto3~=1.10.5
```

Comment:
most of these will already be in setup.py - and any new requirements should go there rather than here in requirements.txt